doc/source/whatsnew/v0.20.0.txt

```diff
@@ -593,6 +593,76 @@ result. On the other hand, this might have backward incompatibilities: e.g.
 compared to numpy arrays, ``Index`` objects are not mutable. To get the original
 ndarray, you can always convert explicitly using ``np.asarray(idx.hour)``.
 
+.. _whatsnew_0200.api_breaking.unique:
+
+pd.unique will now be consistent with extension types
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+In prior versions, using ``Series.unique()`` and ``pd.unique(Series)`` on ``Categorical`` and tz-aware
+datatypes would yield different return types. These are now made consistent. (:issue:`15903`)
+
+- Datetime tz-aware
+
+  Previous behaviour:
+
+  .. code-block:: ipython
+
+     # Series
+     In [5]: Series(pd.Index([pd.Timestamp('20160101', tz='US/Eastern'),
+                              pd.Timestamp('20160101', tz='US/Eastern')])).unique()
+     Out[5]: array([Timestamp('2016-01-01 00:00:00-0500', tz='US/Eastern')], dtype=object)
+
+     In [6]: pd.unique(Series(pd.Index([pd.Timestamp('20160101', tz='US/Eastern'),
+                                        pd.Timestamp('20160101', tz='US/Eastern')])))
+     Out[6]: array(['2016-01-01T05:00:00.000000000'], dtype='datetime64[ns]')
+
+     # Index
+     In [7]: pd.Index([pd.Timestamp('20160101', tz='US/Eastern'),
+                       pd.Timestamp('20160101', tz='US/Eastern')]).unique()
+     Out[7]: DatetimeIndex(['2016-01-01 00:00:00-05:00'], dtype='datetime64[ns, US/Eastern]', freq=None)
+
+     In [8]: pd.unique(pd.Index([pd.Timestamp('20160101', tz='US/Eastern'),
+                       pd.Timestamp('20160101', tz='US/Eastern')]))
+     Out[8]: array(['2016-01-01T05:00:00.000000000'], dtype='datetime64[ns]')
+
+  New Behavior:
+
+  .. ipython:: python
+
+     # Series, returns an array of Timestamp tz-aware
+     Series(pd.Index([pd.Timestamp('20160101', tz='US/Eastern'),
+                      pd.Timestamp('20160101', tz='US/Eastern')])).unique()
+     pd.unique(Series(pd.Index([pd.Timestamp('20160101', tz='US/Eastern'),
+                                pd.Timestamp('20160101', tz='US/Eastern')])))
+
+     # Index, returns a DatetimeIndex
+     pd.Index([pd.Timestamp('20160101', tz='US/Eastern'),
+               pd.Timestamp('20160101', tz='US/Eastern')])
+     pd.unique(pd.Index([pd.Timestamp('20160101', tz='US/Eastern'),
+                         pd.Timestamp('20160101', tz='US/Eastern')]))
+
+- Categoricals
+
+  Previous behaviour:
+
+  .. code-block:: ipython
+
+     In [1]: pd.Series(pd.Categorical(list('aabc'))).unique()
+     Out[1]:
+     [a, b, c]
+     Categories (3, object): [a, b, c]
+
+     In [2]: pd.unique(pd.Series(pd.Categorical(list('aabc'))).unique())
+     Out[2]: array(['a', 'b', 'c'], dtype=object)
+
+  New Behavior:
+
+  .. ipython:: python
+
+     # returns a Categorical
+     pd.Series(pd.Categorical(list('aabc'))).unique()
+     pd.unique(pd.Series(pd.Categorical(list('aabc'))).unique())
```

Comment:
This one (without the `.unique()`, so only with `pd.unique(..)`) still returns something different -> a categorical Series instead of Categorical (like the one above)